banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/utils.py

```diff
@@ -0,0 +1,93 @@
+import frappe
+from pypika.queries import Table
+from frappe.query_builder.functions import CustomFunction
+
+Instr = CustomFunction("INSTR", ["a", "b"])
+RegExpReplace = CustomFunction("REGEXP_REPLACE", ["a", "b", "c"])
+
+# NOTE:
+# Ranking min: 1 (nothing matches), max: 7 (everything matches)
+
+# Types of matches:
+# amount_match: if amount in voucher EQ amount in bank statement
+# party_match: if party in voucher EQ party in bank statement
+# date_match: if date in voucher EQ date in bank statement
+# reference_number_match: if ref in voucher EQ ref in bank statement
+# name_in_desc_match: if name in voucher IN bank statement description
+# ref_in_desc_match: if ref in voucher IN bank statement description
+
+
+def amount_rank_condition(amount: any, bank_amount: float):
```

Comment:
```suggestion
def amount_rank_condition(amount: Column, bank_amount: float) -> Case:
```

I think `any` (as in `any([False, True, False])`) is not a meaningful type.